src/hooks/useNYPLBreakpoints.mdx

```diff
@@ -57,3 +68,81 @@ const MobileLayout = () => ...
 `}
   language="jsx"
 />
+
+## Functional Examples
+
+{BreakPointsExample()}
+
+<Source
+  code={`
+import {
+  Heading,
+  List,
+  Text,
+  useNYPLBreakpoints,
+} from "@nypl/design-system-react-components";
+const {
+  isLargerThanSmall,
+  isLargerThanMedium,
+  isLargerThanLarge,
+  isLargerThanXLarge,
+  isLargerThanSmallMobile,
+  isLargerThanLargeMobile,
+  isLargerThanMobile,
+  isLargerThanSmallTablet,
+  isLargerThanLargeTablet,
+  isLargerThanTablet,
+} = useNYPLBreakpoints();
+
+<Stack direction="row" spacing="l" width="100%">
+  <Box flexBasis="50%">
+    <Heading text="Charka Syntax" level="h3" size="heading6" />
```

Comment:
This is a great visual example. Can you add a description for the user to resize the window to see the values change?

Reply:
Done.